examples/jsm/nodes/shadernode/ShaderNodeUtils.js

```diff
@@ -5,7 +5,7 @@ import SplitNode from '../utils/SplitNode.js';
 import ConstNode from '../core/ConstNode.js';
 import { getValueFromType } from '../core/NodeUtils.js';
 
-const NodeHandler = {
+export const shaderNodeHandler = {
```

Comment:
`shaderNodeHandler` has a constructor, it can be `ShaderNodeHandler`

Reply:
But it is just an object and not a constructor - you can theoretically use `new shaderNodeHandler.constructor()`, but not `new shaderNodeHandler()`.